target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -957,6 +957,10 @@ wzr-hp-g450h)
 xd3200)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$board:blue:wlan2g" "phy1tpt"
 	;;
+xd6700)
+	ucidef_set_led_wlan "status_green" "STATUS GREEN" "$board:green:status" "phy0tpt"
+	ucidef_set_led_wlan "status_red" "STATUS RED" "$board:red:status" "phy1tpt"
```

Comment:
Please, rename these LEDs to wlan2g and wlan5g so that names match functions.
Also, you have "1 x System LED, 1 x LAN / 1 x WAN LED" in your commit description... please, be consistent.